unattended_installer/install_functions/installCommon.sh

```diff
@@ -260,6 +273,29 @@ function installCommon_changePasswords() {
 
 }
 
+function installCommon_checkChromium() {
+
+    if [ "${sys_type}" == "yum" ]; then
+        if (! yum list installed 2>/dev/null | grep -q -E ^"google-chrome-stable"\\.) && (! yum list installed 2>/dev/null | grep -q -E ^"chromium"\\.); then
+            dependencies=(chromium)
```

Comment:
Same use the different variable name.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/2122/commits/614e83b7be2d794699e48f8f16979351918d24f2.